static/styles/left-sidebar.scss

```diff
@@ -125,7 +125,7 @@ li.show-more-topics a {
     width: 100%;
 }
 
-#stream-filters-container .simplebar-content {
+#add-stream-link {
```

Comment:
Could be merged with the existing `#add-stream-link` block below.

Reply:
Fixed!